generators/src/write-rule.ts

```diff
@@ -44,16 +45,8 @@ module.exports = {
       ? {}
       : {
           JSXOpeningElement(node: JSXOpeningElement) {
-            if (
-              node.name.type === "JSXIdentifier" &&
-              componentImports
-                .map((imp) => imp.local.name)
-                .includes(node.name.name)
-            ) {
-              const attribute = node.attributes.find(
-                (attr) =>
-                  attr.type === "JSXAttribute" && attr.name.name === "${propName}"
-              );
+            if (checkMatchingJSXOpeningElement(node, componentImports)) {
```

Comment:
Just a question - do we want to also care about default imports? It was a concern in the `react-component-groups` repo, where I created the `getImportSpecifiersLocalNames` helper for that, which was using the `getDefaultImportsFromPackage` helper inside. But it is rather an exception and most of the rules will still be on the `patternfly-react` repo where we don't have the default imports. So I am ok with keeping the generic generator like this.

Reply:
Yeah it isn't something we need in most cases, but is there any reason we _wouldn't_ want to handle default imports by default though? 🤔